### ISSUE TITLE
G6 is not working with poco f3

### ISSUE THREAD

**Issue by shoctroper** [gamesir123/app#72]
The remap function Is not available.
I'm not able to make it work 
I'm already used the version from Google play and from here and nothing.
Please help I really want to use the controller

**Comment by ChrisnaBadar** [comment]
@shoctroper did you get it to work now? I faced the same problem

**Comment by shoctroper** [comment]
Yes I use the vervsion 4.0.2 of the APK then i donwload the keymapper from the site and runit in dev mode.

when that is done, the remapper will work until the next restar.
